app/src/main/java/com/iic/shopingo/ui/request_flow/activities/SelectShopperActivity.java

```diff
@@ -64,17 +86,25 @@ public View getView(int position, View convertView, ViewGroup parent) {
         itemView = SelectShopperListItemView.inflate(parent);
       }
       itemView.setShopper(getItem(position));
+      if (userLocation != null) {
+        itemView.setUserLocation(userLocation);
+      }
       return itemView;
     }
 
+    private void setUserLocation(Location userLocation) {
+      this.userLocation = userLocation;
+      notifyDataSetChanged();
+    }
+
     // TODO: Move actual model
     public static class Shopper {
       public String photo;
       public String name;
-      public Long latitude;
-      public Long longitude;
+      public Double latitude;
+      public Double longitude;
```

Comment:
Why do you use the objects and not the primitives? (`double` instead of `Double`)
Is it because you want to pass nulls? Is it even a scenario that can happen where the shopper doesn't have coordinates?